src/wallet/rpcwallet.cpp

```diff
@@ -4073,26 +4085,31 @@ static RPCHelpMan send()
                 UniValueType(), // ARR or OBJ, checked later
                 UniValue::VNUM,
                 UniValue::VSTR,
-                UniValue::VOBJ
+                UniValue::VNUM,
+                UniValue::VOBJ,
                 }, true
             );
 
             std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
             if (!wallet) return NullUniValue;
             CWallet* const pwallet = wallet.get();
 
-            UniValue options{request.params[3].isNull() ? UniValue::VOBJ : request.params[3]};
-            if (options.exists("feeRate") || options.exists("fee_rate") || options.exists("estimate_mode") || options.exists("conf_target")) {
-                if (!request.params[1].isNull() || !request.params[2].isNull()) {
-                    throw JSONRPCError(RPC_INVALID_PARAMETER, "Use either conf_target and estimate_mode or the options dictionary to control fee rate");
+            UniValue options{request.params[4].isNull() ? UniValue::VOBJ : request.params[4]};
+            if (options.exists("conf_target") || options.exists("estimate_mode") || options.exists("fee_rate")) {
+                if (!request.params[1].isNull() || !request.params[2].isNull() || !request.params[3].isNull()) {
+                    throw JSONRPCError(RPC_INVALID_PARAMETER, "Use either conf_target and estimate_mode for fee estimation, or fee_rate to specify an explicit fee rate.");
                 }
             } else {
                 options.pushKV("conf_target", request.params[1]);
                 options.pushKV("estimate_mode", request.params[2]);
+                options.pushKV("fee_rate", request.params[3]);
             }
             if (!options["conf_target"].isNull() && (options["estimate_mode"].isNull() || (options["estimate_mode"].get_str() == "unset"))) {
                 throw JSONRPCError(RPC_INVALID_PARAMETER, "Specify estimate_mode");
             }
+            if (options.exists("feeRate")) {
+                throw JSONRPCError(RPC_INVALID_PARAMETER, "Use fee_rate instead of feeRate");
```

Comment:
Optional: mention that `fee_rate` is specified in `[sat/vB]` other than `feeRate`.

Reply:
Done

Reply:
Done in 136234cb